crates/iroha_cli/Cargo.toml

```diff
@@ -46,3 +46,6 @@ futures = { workspace = true }
 [build-dependencies]
 vergen = { version = "8.3.1", default-features = false }
 color-eyre = "0.6.3"
+
+[dev-dependencies]
+iroha_test_network = { workspace = true }
```

Comment:
this is not ok. `iroha_test_network` is for writing integration tests, you can't have it as dependency to other crates